XcodeServerSDK/Server Entities/Device.swift

```diff
@@ -10,21 +10,55 @@ import Foundation
 
 public class Device : XcodeServerEntity {
     
-    public let name: String
-    public let simulator: Bool
     public let osVersion: String
-    public let deviceType: String
     public let connected: Bool
+    public let simulator: Bool
+    public let modelCode: String // Enum?
+    public let deviceType: String // Enum?
+    public let modelName: String
+    public let deviceECID: String?
+    public let modelUTI: String
+    public let activeProxiedDevice: Device?
+    public let docType: String
```

Comment:
`docType` is an internal Xcode Server key used in CouchDB to identify the... surprise surprise... document type. If you want to add it, it should be added to `XcodeServerEntity` instead, because it's also present in entities like bots and integrations.